Formula/restic.rb

```diff
@@ -0,0 +1,30 @@
+class Restic < Formula
+  desc "Backups done right"
+  homepage "https://restic.github.io/"
+  url "https://github.com/restic/restic/archive/v0.7.3.tar.gz"
+  sha256 "6d795a5f052b3a8cb8e7571629da14f00e92035b7174eb20e32fd1440f68aaff"
+  head "https://github.com/restic/restic.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make"
+    bin.install "restic"
+  end
+
+  test do
+    mkdir testpath/"restic_repo"
+    ENV["RESTIC_REPOSITORY"] = testpath/"restic_repo"
+    ENV["RESTIC_PASSWORD"]= "foo"
+
+    (testpath/"testfile").write <<~EOS
```

Comment:
Don't need the `EOS` block here really. `(testpath/"testfile").write "This is a testfile"` is way under 80 characters, so.

Reply:
fixed